backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/dialog/internal/BpkDialogImpl.kt

```diff
@@ -202,6 +218,29 @@ private fun DialogIcon(icon: Dialog.Icon?) {
     }
 }
 
+private fun Modifier.dialogWidth(): Modifier =
+    widthIn(
+        max = DialogMaxWidth,
+    ).padding(
+        top = IconPadding,
+        start = BpkDimension.Spacing.Lg,
+        end = BpkDimension.Spacing.Lg,
+    )
+
+private fun Modifier.contentSize(size: Dp, setSize: (Dp) -> Unit): Modifier =
+    layout { measurable, constraints ->
+        val placeable = measurable.measure(constraints)
+        if (placeable.width < placeable.height) {
+            setSize(placeable.width.toDp())
+        }
+        layout(placeable.width, placeable.height) {
```

Comment:
i think this is where we need to set the correct size (so here for the height we set it to width if the height is larger